account-keys/slip10/src/mnemonic2slip.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import hashlib
+import hkdf
+import hmac
+import mnemonic
+import sys
+
+
+def int_to_string(x, pad):
+    result = ['\x00'] * pad
+    while x > 0:
+        pad -= 1
+        ordinal = x & 0xFF
+        result[pad] = (chr(ordinal))
+        x >>= 8
+    return ''.join(result)
+
+
+def string_to_int(s):
+    result = 0
+    for c in s:
+        if not isinstance(c, int):
+            c = ord(c)
+        result = (result << 8) + c
+    return result
+
+
+# mode 0 - compatible with BIP32 private derivation
+def seed2hdnode(seed, modifier, curve):
+    k = seed
+    while True:
+        h = hmac.new(modifier, seed, hashlib.sha512).digest()
+        key, chaincode = h[:32], h[32:]
+        a = string_to_int(key)
+        if curve == 'ed25519':
+            break
+        if a < curve.order and a != 0:
+            break
+        seed = h
+        # print 'RETRY seed: ' + binascii.hexlify(seed)
+    return key, chaincode
+
+
+def main():
+    args = sys.argv[1:]
+    path_str = ""
+    i = 0
+    for arg in args:
+        print("MnemonicToRistretto {")
+        print(f"    phrase: \"{arg}\",")
+
+        mnemo = mnemonic.Mnemonic("english")
+        seed = mnemo.to_seed(arg, "")
```

Comment:
Re: comment on `seed2hdnode()`, is this where the '2048 iterations' are done?

"We apply 2048 iterations of PBKDF2 with HMAC-SHA512 as the pseudo- random function" (MobileCoin Key Derivation and Recovery section 3.2)

Reply:
Yep